tests/integration/standard/test_application_info.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2025 ScyllaDB, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from cassandra.application_info import ApplicationInfo
+from tests.integration import TestCluster, notscylla, use_single_node, remove_cluster, greaterthanorequalcass40
+
+
+def setup_module():
+    use_single_node()
+
+
+def teardown_module():
+    remove_cluster()  # this test messes with config
+
+
```

Comment:
How does it mess with config? I don't see anything like that here, so please explain.